jormungandr/src/rpc/eth_block_info/logic.rs

```diff
@@ -0,0 +1,55 @@
+use super::Error;
```

Comment:
the module name `jormungandr/src/rpc` is a bit misleading because the node exposes other jrpc api that are not in this module

Reply:
Are you sure that jormungadr node has other `jrpc` ? I have found only `grpc`. 
So maybe rename it into the `jrpc` ?

Reply:
>Are you sure that jormungadr node has other jrpc ? I have found only grpc.

Ah yes, similar sounding letters confused me. 

>So maybe rename it into the jrpc

👍🏼